pkg/app/api/service/apiservice/service.proto

```diff
@@ -69,7 +69,8 @@ message ListApplicationsRequest {
     string name = 1;
     string kind = 2;
     string env_id = 3;
-    bool disabled = 4;
+    string env_name = 4;
+    bool disabled = 5;
```

Comment:
The index number should not be changed to keep the backward-compatibility.
(We the control-plane is deployed with this version, the old versions of pipectl are still running.)

Reply:
🙆‍♀️